webui/src/components/editrecord.jsx

```diff
@@ -168,7 +168,14 @@ const EditRecord = React.createClass({
             }
         }
         console.assert(!Array.isArray(value));
-        r = value !== undefined ? r.setIn(path, value) : r.deleteIn(path);
+        let trimmedValue;
```

Comment:
Please just change the default `value` here and not add this `trimmedValue` variable. It's less code and less buggy.